client/my-sites/sharing/main.jsx

```diff
@@ -1,90 +1,95 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:my-sites:sharing' ),
-	find = require( 'lodash/find' );
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { find, get } from 'lodash';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var SectionNav = require( 'components/section-nav' ),
-	NavTabs = require( 'components/section-nav/tabs' ),
-	NavItem = require( 'components/section-nav/item' ),
-	Main = require( 'components/main' ),
-	SidebarNavigation = require( 'my-sites/sidebar-navigation' ),
-	utils = require( 'lib/site/utils' ),
-	sites = require( 'lib/sites-list' )();
-
+import { canCurrentUser } from 'state/selectors';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionNav from 'components/section-nav';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
 import UpgradeNudge from 'my-sites/upgrade-nudge';
 
-module.exports = React.createClass( {
-	displayName: 'Sharing',
-
-	componentDidMount: function() {
-		debug( 'Sharing React component mounted.' );
-	},
-
-	getSelectedText: function() {
-		var selected = find( this.getFilters(), { path: this.props.path } );
-		if ( selected ) {
-			return selected.title;
-		}
-
-		return '';
-	},
-
-	getFilters: function() {
-		var site = sites.getSelectedSite(),
-			pathSuffix = sites.selected ? '/' + sites.selected : '',
-			filters = [];
+class Sharing extends Component {
```

Comment:
Given it has no state, would probably be fairly easy to express this as a function component if we're comfortable collapsing `getFilters` into the `render` logic.